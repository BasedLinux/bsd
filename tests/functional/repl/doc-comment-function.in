:doc import ./doc-comment-function.bsd
