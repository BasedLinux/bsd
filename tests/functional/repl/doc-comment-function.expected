Bsd <bsd version>
Type :? for help.

bsd-repl> :doc import ./doc-comment-function.bsd
Function defined at /path/to/tests/functional/repl/doc-comment-function.bsd:4:1

 
A doc comment for a file that only contains a function
